src/assets/crisistextline/sass/partials/02-elements/_icons.scss

```diff
@@ -34,18 +34,6 @@
     text-align: center;
 }
 
-$user-icon-baby-chick: "🐣";
-$user-icon-year-1: "🌟";
-$user-icon-year-2: "💫";
-$user-icon-year-3: "✨";
-$user-icon-year-4: "⚡️";
-$yearStarList: $user-icon-year-1 $user-icon-year-2 $user-icon-year-3 $user-icon-year-4;
-@mixin counselor-year-star($year, $element) {
```

Comment:
Looks like `counselor-year-star` and `user-icon-baby-chick` are used in _sidebar.scss, so you'll probably need to remove those references too. I'm not sure how Sass deals with missing mixins/variables, but I'd guess it doesn't do great (and would be good to clean up even if it's not strictly required)

Reply:
@morrismatts - Okay, just took that out if you want to take another look. Thanks!